0.17/files/docker-entrypoint.sh

```diff
@@ -47,15 +47,15 @@ if [ "$(id -u)" = '0' ]; then
   SU_EXEC="su-exec factorio"
```

Comment:
If this if-block doesn't get executed, then SU_EXEC is undefined and will cause an error. It should either be set to an empty string at the start of the script, or there should be an else block setting it to one.